sw/device/lib/dif/BUILD

```diff
@@ -0,0 +1,34 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+cc_library(
+  name = "clkmgr",
+  srcs = [
+      ":dif_clkmgr.c",
+      ":dif_clkmgr.h",
+      ":dif_warn_unused_result.h",
+      ],
+  deps = [
+      "//sw/device/lib/base:mmio",
+      "//hw/top_earlgrey/ip/clkmgr/data/autogen:clkmgr_regs",
```

Comment:
I'm worried this could cause problems if building for englishbreakfast.

Reply:
I think the way meson currently builds the autogen headers we'd have the same problem.  We can tackle this problem at some later time with a `select` rule for choosing the correct headers.